.travis.yml

```diff
@@ -3,10 +3,17 @@ language: php
 php:
     - 5.5
     - 5.6
+    - 7.0
     - hhvm
 
+matrix:
+    include:
+        - php: 5.5
+          env: 'COMPOSER_FLAGS="--prefer-stable --prefer-lowest"'
+
 before_script:
-    - travis_retry composer install --no-interaction
+    - travis_retry composer self-update
```

Comment:
Ultimately, this isn't worth getting worked up over. This is sooo trivial.

If you want to self-update composer on your project, then do it. The world isn't going to implode. I'm only recommending that you don't do it.